net/siproxd/src/opnsense/mvc/app/views/OPNsense/Siproxd/general.volt

```diff
@@ -0,0 +1,147 @@
+
+#}
+<!-- Navigation bar -->
+<ul class="nav nav-tabs" data-tabs="tabs" id="maintabs">
+    <li class="active"><a data-toggle="tab" href="#general">{{ lang._('General') }}</a></li>
+    <li><a data-toggle="tab" href="#users">{{ lang._('Users') }}</a></li>
+    <li><a data-toggle="tab" href="#domains">{{ lang._('Outbound Domains') }}</a></li>    
+    <li><a data-toggle="tab" href="#showregistrations">{{ lang._('Current registrations') }}</a></li>
+</ul>
+    
+<div class="tab-content content-box tab-content">
+    <div id="general" class="tab-pane fade in active">
+        <div class="content-box" style="padding-bottom: 1.5em;">
+            {{ partial("layout_partials/base_form",['fields':generalForm,'id':'frm_general_settings'])}}
+            <hr />
+            <div class="col-md-12">
+                <button class="btn btn-primary"  id="saveAct" type="button"><b>{{ lang._('Save') }}</b><i id="saveAct_progress" class=""></i></button>
+            </div>
+        </div>
+    </div>
+    <div id="users" class="tab-pane fade in">
+        <table id="grid-users" class="table table-responsive" data-editDialog="dialogEditSiproxdUser">
+            <thead>
+                <tr>
+                    <th data-column-id="enabled" data-type="string" data-formatter="rowtoggle">{{ lang._('Enabled') }}</th>
+                    <th data-column-id="username" data-type="string" data-visible="true">{{ lang._('Username') }}</th>
+                    <th data-column-id="password" data-type="string" data-visible="true">{{ lang._('Password') }}</th>
+                </tr>
+            </thead>
+            <tbody>
+            </tbody>
+            <tfoot>
+                <tr>
+                    <td colspan="5"></td>
+                <td>
+                    <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
+                    </td>
+                </tr>
+            </tfoot>
+        </table>
+    </div>
+    <div id="domains" class="tab-pane fade in">
+        <table id="grid-domains" class="table table-responsive" data-editDialog="dialogEditSiproxdDomain">
+            <thead>
+                <tr>
+                    <th data-column-id="enabled" data-type="string" data-formatter="rowtoggle">{{ lang._('Enabled') }}</th>
+                    <th data-column-id="name" data-type="string" data-visible="true">{{ lang._('Name') }}</th>
+                    <th data-column-id="host" data-type="string" data-visible="true">{{ lang._('Host') }}</th>
+                    <th data-column-id="port" data-type="string" data-visible="true">{{ lang._('Port') }}</th>
+                </tr>
+            </thead>
+            <tbody>
+            </tbody>
+            <tfoot>
+                <tr>
+                    <td colspan="5"></td>
+                <td>
+                    <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
```

Comment:
this two lines are missing an indent step

Reply:
fixed